modules/eventing/pages/eventing-language-constructs.adoc

```diff
@@ -382,49 +346,102 @@ _Example iterator using a named params object_
     );
 ----
 
-* _options_
-+
-This is a JSON object having various query runtime options as keys. Currently, the following settings are recognized:
+|`options`
+|A JSON object that has various query runtime options as keys.
+The following settings are available:
+
+`isPrepared` determines if the statement is prepared.
+This setting defaults to `false`, but you can change it to `true` to increase the performance of any {sqlpp} query.
+
+`consistency` determines the consistency level for the statement.
+This setting defaults to the consistency level specified in your Eventing Function settings, but you can change it on any individual statement.
+Valid values are `none` and `request`.
+
+|`return value (handle)`
+|Returns a JavaScript iterable object that represents the result set of the query.
+You can iterate the returned handle using standard JavaScript mechanisms like `for...of` loops.
+
+You can use the `close()` method on the handle object to release the resources held by the {sqlpp} query.
+This method also cancels queries that are in the process of streaming results.
+
+|Exceptions thrown
+|The `N1QL()` function throws an exception if the underlying {sqlpp} query fails to parse or does not start to execute.
+
+The returned iterable handle throws an exception if the underlying {sqlpp} query fails after it has started.
+
+The `close()` method on the iterable handle can throw an exception if the underlying {sqlpp} query cancellation finds an unexpected error.
+
+|===
+
+[#analytics_call]
```

Comment:
Hi @jon-strabala, this section on the ANALYTICS() Function Call is the only new content on the page. Everything else is just reworked existing content.